README.md

```diff
@@ -13,7 +13,8 @@ Currently supported data sources are:
 * SQL
 
 Sematext App Agent uses Influx Line Protocol to ship the metrics. The metrics collected by the agent can be shipped to
-any Influx Line Protocol complaint endpoints like InfluxDB. You can also add support for other output formats like HTTP, Graphite, etc.
+any Influx Line Protocol complaint endpoints like InfluxDB. In future, you can also add support for other output formats
```

Comment:
complaint => compliant
In future, you can also add => In the future, we will add